module/polls/helper.js

```diff
@@ -0,0 +1,631 @@
+/**
+ * name : polls/helper.js
+ * author : Deepa
+ * created-date : 01-Aug-2020
+ * Description : Polls helper functionality.
+ */
+
+// Dependencies
+const formsHelper = require(MODULES_BASE_PATH + "/forms/helper");
+const appsPortalBaseUrl = process.env.APP_PORTAL_BASE_URL ? process.env.APP_PORTAL_BASE_URL : "https://apps.shikshalokam.org/";
```

Comment:
@deepakr123 - Check exists and not blank condition.

Reply:
resolved